src/detect-flowbits.c

```diff
@@ -182,6 +182,20 @@ static int DetectFlowbitParse(char *str, char *cmd, int cmd_len, char *name,
             SCLogError(SC_ERR_PCRE_GET_SUBSTRING, "pcre_copy_substring failed");
             return 0;
         }
+
+        /* Trim trailing whitespace. */
+        while (isblank(name[strlen(name) - 1])) {
```

Comment:
Done in new PR.

> Should this be applied to hostbits & xbits as well? And I guess to flowint and vars and such?

New PR has it done for hostbots.  The others weren't as straight forward as hostbits and flowbits.